### PR TITLE
Update gradle to 3.3

### PR DESCRIPTION
**Pull request by beeender**

### CODE REVIEW

**Comment by beeender** [comment]
@realm/java

File: gradle/wrapper/gradle-wrapper.properties [hunk 1]
```diff
@@ -1,6 +1,6 @@
-#Wed Oct 05 09:59:35 CEST 2016
+#Fri Mar 17 16:04:27 CST 2017
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-3.1-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-3.3-all.zip
```
Comment (cmelchior) [inline gradle/wrapper/gradle-wrapper.properties]: Isn't 3.4.1 latest version?

Comment (beeender) [inline gradle/wrapper/gradle-wrapper.properties]: Android Studio suggests to sue 3.3, shall we stay with what AS suggests?

Comment (cmelchior) [inline gradle/wrapper/gradle-wrapper.properties]: @bmeike Updated the realm-java one to 3.4.1, so I would probably just use that?

**Pull request merged**